TcUnit-Runner/Program.cs

```diff
@@ -58,8 +59,8 @@ static void Main(string[] args)
                 .Add("v=|VisualStudioSolutionFilePath=", "The full path to the TwinCAT project (sln-file)", v => VisualStudioSolutionFilePath = v)
                 .Add("t=|TcUnitTaskName=", "[OPTIONAL] The name of the task running TcUnit defined under \"Tasks\"", t => TcUnitTaskName = t)
                 .Add("a=|AmsNetId=", "[OPTIONAL] The AMS NetId of the device of where the project and TcUnit should run", a => AmsNetId = a)
-                .Add("?|h|help", h => showHelp = h != null);
-
+                .Add("?|h|help", h => showHelp = h != null)
+                .Add("tc=|TwinCATVersion=", "[OPTIONAL] The TwinCAT version to be used to load the TwinCAT project", tc => forceToThisTwinCATVersion = tc);
```

Comment:
Use uppercase F to be consistent with TcUnitTaskName and AmsNetId

Reply:
okay